node/node_handler.go

```diff
@@ -400,11 +390,11 @@ func (node *Node) PostConsensusProcessing(newBlock *types.Block) error {
 			for _, addr := range node.GetAddresses(newBlock.Epoch()) {
 				wrapper, err := node.Beaconchain().ReadValidatorInformation(addr)
 				if err != nil {
-					return err
+					return nil
```

Comment:
why ignore the error here? do you want to print a warning?

Reply:
it's for webhook, don't want to mess up normal consensus logic by returning error. Added error log.